compiler/tests-spec/testData/diagnostics/s-16.30_when-expression/p-2/neg/3.1.kt

```diff
@@ -0,0 +1,15 @@
+// SKIP_TXT
```

Comment:
Maybe we could skip txt generation for all these tests automatically by adding an abstract method into `AbstractDiagnosticsTest` and overriding it in this test, instead of adding `SKIP_TXT` to each file?

Reply:
Good idea.
In `AbstractDiagnosticsTest` already exist similar abstract method — `skipDescriptorsValidation`. I just used it in `AbstractDiagnosticsTestSpec`.
See: 8dcbd8074d6814218e1e16c373cb41e552a5942a

Reply:
Thanks. There's an empty line at the beginning of each file now though 😀

Reply:
Exactly! Thanks. aa16d88693d212e37782c138601e3c78e1ad23a2